scripts/deploy.js

```diff
@@ -0,0 +1,93 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+const { ethers } = require("hardhat");
+const fs = require('fs')
+const { NFTStorage, File } = require('nft.storage')
+const dotenv = require('dotenv');
+const { MerkleTree } = require('merkletreejs')
+const keccak256 = require('keccak256')
+dotenv.config();
+
+const whitelistAddresses = [
```

Comment:
More a general comment, are we following a style guide? Deeply immutable constants would generally be `CONSTANT_CASE` in the Google JS Guide: https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
It's always good to stick to style guides, so I will change that.